spec/models/gadget_spec.rb

```diff
@@ -8,12 +8,25 @@
   it { is_expected.to be_versioned }
 
   describe "updates", versioning: true do
-    it "generates a version for updates to `name` attribute" do
+    it "generates a version for updates" do
       expect { gadget.update_attribute(:name, "Hammer") }.to(change { gadget.versions.size }.by(1))
     end
 
-    it "ignores for updates to `brand` attribute" do
-      expect { gadget.update_attribute(:brand, "Stanley") }.not_to(change { gadget.versions.size })
+    context "ignored via symbol" do
+      it "doesn't generate a version" do
+        expect { gadget.update_attribute(:brand, "Picard") }.not_to(change { gadget.versions.size })
```

Comment:
Updated name to `Picard` so the line doesn't exceed the maximum for Rubocop.

Reply:
That's fine. You can also break long lines like this:

```suggestion
        expect { 
          gadget.update_attribute(:brand, "Stanley") 
        }.not_to(change { gadget.versions.size })
```

Reply:
Sounds good!